backend/src/services/episode/episode.service.ts

```diff
@@ -245,6 +245,13 @@ class Episode {
 
     return episode;
   }
+
+  public async deleteAllByPodcastId(id: number): Promise<void> {
+    const episodes = await this.#episodeRepository.getAllByPodcastId(id);
+    episodes.forEach((episode) => {
+      this.delete(episode.id);
+    });
```

Comment:
```suggestion
    episodes.forEach(async (episode) => {
      await this.delete(episode.id);
    });
```
We should await before all episodes will be deleted and only after move on